pkg/cli/admin/catalog/mirror.go

```diff
@@ -71,7 +71,10 @@ var (
 	`)
 )
 
-const IndexLocationLabelKey = "operators.operatorframework.io.index.database.v1"
+const (
+	IndexLocationLabelKey = "operators.operatorframework.io.index.database.v1"
+	mbSize                = 1000000
```

Comment:
Nit: Should this just be `maxBites`? mb is confusing because my first thought is megabytes

Reply:
I'll remove it as requested in https://github.com/openshift/oc/pull/818/files#r627375702